life/updated/02-life.adoc

```diff
@@ -491,96 +491,96 @@ but blessed be the name and power of the Lord for his infinite mercies,
 according to the day, so was our strength.
 
 A little time after,
-the officers came and demanded money for the king for our meeting together.
+the officers came and demanded money for the king because of our meeting together.
 My husband answered them, "`If I owed the king any, I would surely pay him;
 but seeing I owe him no money, I will pay him none.`"
-They asked leave to distrain his goods; to which he said, "`If you will take my goods,
+They asked leave to seize his goods; to which he said, "`If you will take my goods,
 I cannot hinder you, but I will not give you leave to take them;
-neither will I be accessary to your taking them.`"
-The officers seeing our innocency, for we were in our shop at our lawful calling,
-with our hands to our labor, and our children with us,
-the constable leaned his head down upon his hand, with a heavy heart, and said,
+nor will I be accessary to your taking them.`"
+The officers saw our innocence, for we were in our shop at our lawful calling,
+with our hands to our labor, and our children with us.
+Then the constable leaned his head down upon his hand, with a heavy heart, and said,
 "`It is against my conscience to take their goods from them.`"
-Then I said, "`John, have a care of wronging your conscience;
-for what could the Lord do more for you than to place his good spirit in your heart,
-to teach you what you should do, and what you should leave undone.`"
+Then I said, "`John, have a care of wronging your conscience,
+for what could the Lord do more for you than to place His good spirit in your heart,
+to teach you what you should do, and what you should leave undone?`"
 He said, "`I know not what to do in this matter; if paying the money once would do,
-I would do it, but it will not end so; it will be thus, while you keep going to meeting;
-for the rulers have made such laws, that never was the like in any age.`"
+I would do it, but it will not end so,
+it will continue to be thus while you keep going to meeting,
+for the rulers have made such laws that never was the like in any age.`"
 I said, "`John, when you have wronged your conscience,
 and brought a burden upon your spirit, it is not the rulers who can remove it from you.
-If you should go to the rulers, and say,
-I have done that which was against my conscience to do,
-they may say as the rulers did to Judas, What is that to us, see you to that.`"
+If you should go to the rulers and say,
+'`I have done that which was against my conscience to do,`'
+they may say as the rulers did to Judas, '`What is that to us, see you to that.`'`"
 
-The officers who were with him, came and pulled down our goods;
+The officers who were with him came and pulled down our goods,
 and the power of the Lord smote them,
-insomuch that paleness was in their faces and their lips quivered,
+so that paleness was in their faces and their lips quivered,
```

Comment:
Does insomuch sound really weird to you? Maybe I'm just used to it... but I kinda feel like it still works ok.  I'll leave it up to you.